src/quo2/components/counter/counter/style.cljs

```diff
@@ -12,19 +12,16 @@
 
 (defn container
   [{:keys [type label container-style customization-color theme value max-value]}]
-  (let [label-length (count label)
-        width        (case label-length
-                       1 16
-                       2 20
-                       28)]
+  (let [width (case (count label)
+                1 16
+                2 20
+                28)]
     (cond-> (merge
-             {:align-items       :center
-              :justify-content   :center
-              :border-radius     6
-              :width             width
-              :margin-vertical   2
```

Comment:
I removed the `margin-vertical` and `margin-horizontal` added here:
https://github.com/status-im/status-mobile/commit/dfa30fd14fd6af2e2882455bd877fb92557c34bf

and added that style to channel component using `container-style` prop instead. Otherwise it clashes with other uses of this component